### PR TITLE
[devtools-translate] Spanish - What's New in DevTools (Chrome 99) 

### LINKED ISSUE #2150: [devtools-translate] Spanish - What's New in DevTools (Chrome 99)
Post: https://developer.chrome.com/blog/new-in-devtools-99/

Translator: @midudev
Reviewer: @Caballerog

Translated by: Feb 25

### PR DESCRIPTION
**Pull request by midudev**
Fixes #2150 

Changes proposed in this pull request:

- Add Spanish translation for What's New Chrome 99
- Fix perfers typo on English article and other comments for next translations.

### CODE REVIEW

**Comment by chybie** [comment]
Thanks @midudev for the review & spotting the `prefers` typo! Over to you @Caballerog .

File: site/es/blog/new-in-devtools-99/index.md [hunk 1]
```diff
@@ -5,190 +5,222 @@ authors:
...
-<!-- 1. Remove the "draft: true" tag above when submitting PR -->
-<!-- 2. Provide translations under each of the English commented original content -->
-<!-- 3. Translate the "description" tag above -->
-<!-- 4. Translate all the <img> alt text -->
-<!-- 5. Update the whats-new.md file -->
-<!-- end: translation instructions -->
-
 <!-- ## Throttling WebSocket requests {: #websocket } -->
-
+## Regulación de las peticiones WebSocket {: #websocket }
```
Comment (Caballerog) [inline site/es/blog/new-in-devtools-99/index.md:22]: Perhaps this concept is better understood using the translation "Limitación"? What do you think?

File: site/es/blog/new-in-devtools-99/index.md [hunk 2]
```diff
@@ -5,190 +5,222 @@ authors:
...
 <!-- The **Network** panel now supports throttling web socket requests. Previously, the network throttling didn't work on web socket requests. -->
-
+El panel **Red** ahora permite regular peticiones web socket. Antes, la regulación de las peticiones de la red no funcionaba en peticiones web socket.
```
Comment (Caballerog) [inline site/es/blog/new-in-devtools-99/index.md:24]: limitar instead of regular

**Review by Caballerog — COMMENTED** [review]
Perfect, just a slight debate on a word

**Comment by midudev** [comment]
Fixed @Caballerog 👍

**Review by and-oli — APPROVED** [review]
LGTM! Muchas gracias, Miguel Ángel!

**Comment by chybie** [comment]
Gracias both!

**Pull request merged**